classes/class-loader.php

```diff
@@ -330,7 +330,7 @@ public function enqueue_editor_scripts( $hook ) {
 
 		// Enqueue editor script for all Blocks in this Plugin.
 		self::enqueue_local_script(
-			'planet4-blocks-script',
+			'planet4-blocks-editor-script',
```

Comment:
I think it's sane to change the handles if we are going to make it closer to WP's conventions, as the change to `edit.js` and so on.

Reply:
It's definitely a good idea, not arguing with that. My suggestion is to separately rename in a follow up PR, with the goal of reducing the amount of risk for this PR, since we're already changing a lot.

Reply:
Sure, I get the risk, but I think this handle is not used broadly, right? We can do a quick search in the blocks repo and the master theme and that's it. Maybe in some customized child-themes in case someone added it as a dependency, but other than that, this change was meant to make it consistent with the rest of the naming conventions in this PR, not changing seems like a flaw to me.

Reply:
From a search on Github it looks like this repo is the only one using this handle.

Reply:
Yeah I guess if we're just renaming the handle and nothing else is using it, it should be fine :ok_hand: